### PR TITLE
community/emacs cleaned up testing&unmaintained, bumped v to 25.1, ad…

### PR DESCRIPTION
**Pull request by stef**
…ded nox, x11 and gtk2 variants

### CODE REVIEW

**Comment by stef** [comment]
this PR supersedes #367

**Comment by clandmeter** [comment]
shouldn't hicolor-icon-theme desktop-file-utils be part of the subpkg depends?

**Comment by clandmeter** [comment]
Looks good now! Can you squash them?

File: community/emacs/APKBUILD [hunk 1]
```diff
@@ -0,0 +1,153 @@
...
+	ncurses-libs gnutls-dev libxaw-dev"
+subpackages="$pkgname-doc $pkgname-nox $pkgname-x11 $pkgname-gtk2"
+source="ftp://ftp.gnu.org/gnu/emacs/emacs-$pkgver.tar.xz
+	gamedir.patch
+	noaslr-dump.patch
+	musl.patch"
+
+prepare() {
+	cd "$builddir/$1"
+   default_prepare || return 1
```
Comment (albap) [inline community/emacs/APKBUILD]: wrong indent

**Pull request closed without merging**